examples/Cargo.toml

```diff
@@ -9,7 +9,7 @@ publish = false
 
 [dependencies]
 bytes = "0.5.6"
-pb-jelly = "0.0.5"
+pb-jelly = { path = "../pb-jelly" }
```

Comment:
Per https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations - you can actually specify both path and version

Actually, in this case, since examples aren't being posted to crates.io, we probably should keep it as only a path dependency.